Cargo.toml

```diff
@@ -19,8 +19,9 @@ version = "0.7"
 features = ["extension-module"]
 
 [dependencies]
+itertools = "0.8"
 failure = "0.1"
-finalfusion = "0.7"
+finalfusion = { git = "https://github.com/finalfusion/finalfusion-rust.git" }
```

Comment:
Pin it to a specific revision to avoid build failures when incompatible changes are made in `finalfusion-rust` (`rev = "somerevision"`).